cognite/client/data_classes/workflows.py

```diff
@@ -238,6 +243,49 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         return output
 
 
+class SimulationTaskParameters(WorkflowTaskParameters):
+    """
+    The simulation parameters are used to specify the simulation routine to be executed.
+    Args:
+        routine_external_id (str): The external ID of the simulation routine to be executed.
+        run_time (int | None): Reference timestamp used for data pre-processing and data sampling.
+        inputs (list[SimulationInputOverride] | None): List of input overrides.
+    """
+
+    task_type = "simulation"
+
+    def __init__(
+        self,
+        routine_external_id: str,
+        run_time: int | None = None,
```

Comment:
These are missing from the spec. Can you please update the API docs?

Reply:
https://api-docs.cognite.com/20230101-beta/tag/Workflow-versions/operation/CreateOrUpdateWorkflowVersion
<img width="907" alt="Screenshot 2024-12-01 at 10 49 23" src="https://github.com/user-attachments/assets/d323ad97-3f1d-401e-b59c-0451863ce969">
They're here, not sure what you mean?